### DISCUSSION TITLE
Content item creation by Anonymous

### DISCUSSION THREAD

**Discussion by barbababak** [OrchardCMS/OrchardCore#16857] [General]
Is there a way that someone other than the admin (Anonymous) can create a content item such as ContactUs?

**Comment by Piedone** [comment]
The short answer is yes, but what exactly are you looking for?

For something like a contact form, you could post the form to a workflow, and create content items from form submission in the workflow.

**Comment by hishamco** [comment]
I already did that a few days ago in a contact form using workflow as @Piedone mentioned above, you could use `Create Content Task`, then pass the `Request.XXX` that comes from the form into the task content as Liquid

**Comment by barbababak** [comment]
I can almost do this in the admin dashboard... but outside of the dashboard, how can I show a form to "create" a new content item or "edit" an existing user content item to a non-admin user using a workflow? And what is the "action" of the form?

**Reply by hishamco** [reply]
Just a normal HTML form with post action

**Comment by barbababak** [comment]
I have been searching for several days and have not found an answer
Is there a way in Orchard Core to display the same form of creating a content item for a content type that is displayed to the admin in the dashboard section for a normal user (authenticated or anonymous) and the rest of the work (saving the content item is also like the dashboard Can it be done by orchard core?
If there is an educational resource on the Internet, please introduce it
For another part, I need the user (authenticated) to be able to edit his previous content items ...

**Reply by Piedone** [reply]
Yes, but you'll have to code for it. See [this controller](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs) that handles the admin content editor for some inspiration, and see [this other](https://github.com/Lombiq/Orchard-Training-Demo-Module/blob/dev/Lombiq.TrainingDemo/Controllers/PersonListController.cs) heavily documented example of managing content items from code.
